src/printer.ts

```diff
@@ -1130,7 +1210,7 @@ export class PugPrinter {
 				break;
 		}
 		result += token.mustEscape ? '#' : '!';
-		result += `{${token.val}}`;
+		result += handleBracketSpacing(this.options.pugBracketSpacing, token.val.trim(), ['{', '}']);
```

Comment:
I accidentally found out that we could support bracket spacing for `interpolated-code` tokens :astonished:  
Maybe we should create it's own PR so we can squash-merge this and have a better history in `main` branch

Reply:
Extracted this to its own PR in #208